app/Resources/views/repository/layout.html.twig

```diff
@@ -4,12 +4,17 @@
 
 {% block content %}
     <span class="pull-right">
-            {{ macros.repository_status(repository) }}
-        </span>
+        {{ macros.repository_status(repository) }}
+    </span>
 
     <ol class="breadcrumb">
         <li><a href="{{ path('account', {account: repository.owner.username}) }}">{{ repository.owner.username }}</a></li>
-        <li class="active">{{ repository.name }}</li>
+        <li class="active">
+            {{ repository.name }}
+            {% if is_granted('ROLE_USER') %}
+                {{ macros.repository_star(repository, is_starred) }}
```

Comment:
What do you of making  `is_starred` a twig function ?
If so, it will simplify the controller.